snd/test/src/org/labkey/test/tests/snd/SNDTest.java

```diff
@@ -2190,13 +2200,96 @@ public void categoryPermissionsUI() throws Exception
 
         for (int j = 0; j < categoryRows.size(); j++)
         {
-            value = getPerimissionTableValue(categoryRows.get(j), 1);
+            value = getPermissionTableValue(categoryRows.get(j), 1);
             assertNotNull(value);
             assertTrue(value.equals(permissions.get(j)));
         }
 
     }
 
+    @Test
+    public void testQueryProvisionedSnapshot()
+    {
+        String SOURCE_QUERY_NAME = "SND: Source query";
+        String SNAPSHOT_NAME = SOURCE_QUERY_NAME + " Snapshot";
+        String SOURCE_QUERY = "SELECT\n" +
+                "lsid AS _key,\n" +
+                "SubjectId AS participantid,\n" +
+                "date,\n" +
+                "qcstate,\n" +
+                "lsid,\n" +
+                "amount,\n" +
+                "units,\n" +
+                "kit_type\n" +
+                "FROM SND.Categories.Sodium\n";
+
+        String UPDATED_SOURCE_QUERY = "SELECT\n" +
+                "lsid AS _key,\n" +
+                "SubjectId AS participantid,\n" +
+                "date,\n" +
+                "qcstate,\n" +
+                "lsid,\n" +
+                "'8976' AS amount,\n" +
+                "units,\n" +
+                "kit_type\n" +
+                "FROM SND.Categories.Sodium";
```

Comment:
Consider referencing the original query to make it more clear what is changing.
```suggestion
        String UPDATED_SOURCE_QUERY = SOURCE_QUERY.replace(
                "amount,",
                "'8976' AS amount,");
```